src/vnsw/agent/vrouter/ksync/test/test_vnswif.cc

```diff
@@ -117,9 +117,11 @@ TEST_F(TestVnswIf, intf_inactive) {
 
 // Validate that link-local address is deleted when interface is deleted
 TEST_F(TestVnswIf, intf_delete) {
+	Ip4Address vnet1_address = vnet1_->mdata_ip_addr();
     DeleteVmportEnv(input, 1, false);
+    vnet1_ = 0;
     client->WaitForIdle();
-    EXPECT_FALSE(vnswif_->IsValidLinkLocalAddress(vnet1_->mdata_ip_addr()));
+    EXPECT_FALSE(vnswif_->IsValidLinkLocalAddress(vnet1_address));
```

Comment:
also why did you make this change?

Reply:
That's because vnet1_ doesn't exist anymore.